lcoe_calculator/tech_processors.py

```diff
@@ -231,7 +231,7 @@ class CoalProc(TechProcessor):
     ]
 
     sheet_name = 'Coal_FE'
-    num_tds = 4
+    num_tds = 5
```

Comment:
IIRC, `num_tds` is a bit of a hack as it's defined in the xlsx, but I couldn't figure out an effective way to figure it out w/ python. Something we could look into fixing in the future.